app/models/katello/content_view_version.rb

```diff
@@ -184,6 +185,18 @@ def products(env = nil)
       end
     end
 
+    def exporter_name
```

Comment:
From what I can tell these 3 methods seem useful only in the pulp3 context. If we have a place, I think they could go into one of the pulp3 service classes, or perhaps stay in the action where used

Reply:
Yeh but they belong to the version. I can try creating a "app/services/katello/pulp3/content_view_version.rb" and have it included.

Reply:
I thought about this and feel the pulp3 separation as unnecessary given this PR  is going to only work with pulp3 anyway

Reply:
AFAIK we dont have the logic to say "include this module only if pulp3" in any of the models . @jlsherrill correct me if wrong.

Reply:
we don't, but a service class wouldn't be a module that you include anyways. It would be its own object potentially, something like:
```
exporter = Katello::Pulp3::Exporter.new(cvv)
exporter.generate_exporter_id
expoter.exporter_id
```

Reply:
updated. Created a service class -> `app/services/katello/pulp3/content_view_version.rb` where all this logic is handled.